js/tji-datasets-explore.js

```diff
@@ -685,6 +685,12 @@ TJIChartView.prototype.create_chartview_DOM = function() {
 // attach delegated event handlers to parent DOM elements that are data agnostic
 TJIChartView.prototype.attach_events = function() {
   var that = this;
+
+	// Expand filter panel by default on larger screens
+	if (jQuery(window).width() > 768) {
```

Comment:
since I'm variable crazy -- think you could make 768 a breakpoint variable/constant -- kinda like we have color variables at the top. Something like 
```
var BREAKPOINTS = { 
  sm: '768',
  md: '1024',
  lg: '1200',
};
```